factory_bot_rails.gemspec

```diff
@@ -1,20 +1,20 @@
 Gem::Specification.new do |s|
-  s.name        = %q{factory_bot_rails}
-  s.version     = '4.11.1'
+  s.name        = "factory_bot_rails"
+  s.version     = "4.11.1"
   s.authors     = ["Joe Ferris"]
-  s.email       = %q{jferris@thoughtbot.com}
+  s.email       = "jferris@thoughtbot.com"
   s.homepage    = "https://github.com/thoughtbot/factory_bot_rails"
-  s.summary     = %q{factory_bot_rails provides integration between
-    factory_bot and rails 3 or newer}
-  s.description = %q{factory_bot_rails provides integration between
-    factory_bot and rails 3 or newer (currently just automatic factory definition
-    loading)}
+  s.summary     = "factory_bot_rails provides integration between "\
+                  "factory_bot and rails 3 or newer"
+  s.description = "factory_bot_rails provides integration between "\
+                  "factory_bot and rails 3 or newer (currently just automatic "\
+                  "factory definition loading)}"
```

Comment:
We ended up with one extra `}`, but other than that this all looks good to me!

Reply:
oy, good catch @composerinteralia  😬 
fixed here: https://github.com/thoughtbot/factory_bot_rails/pull/295/commits/ac5432d9eff7413f046a38c6d033678e4879e294